server_management/management/commands/pushdb.py

```diff
@@ -3,88 +3,89 @@
 from django.core.management import call_command
 from fabric.api import env, local, run, settings, sudo
 
-from ._core import ServerManagementBaseCommand, load_config, run_tasks
+from ._core import ServerManagementBaseCommand, load_config, run_tasks, can_push
 from .backupdb import perform_backup
 
 class Command(ServerManagementBaseCommand):
 
     def handle(self, *args, **options):
-        # Load server config from project
-        config, remote = load_config(env, options.get('remote', ''), debug=options.get('debug', False))
+        if can_push():
```

Comment:
This overly complicates the diff; the _simplest_ case should be tested first. Check the negative case of `can_push` here and raise an exception if it is falsey. This reduces the size of the guarded branch.